src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -1,65 +1,201 @@
+                        <i class="bi bi-pencil-square"></i>
+                    </button> <!-- TODO: button needs to be added properly -->
+                </td>
+            </tr>
         }
-    </div>
-</div>
+        </tbody>
+    </table>
 
-<!-- pagination -->
-<nav aria-label="Page navigation example">
-    <ul class="pagination">
-        <li class="page-item disabled">
-            <button class="page-link">
-                🡸
-            </button>
-        </li>
-        <li class="page-item active"
-            aria-current="page">
-            <button class="page-link">
-                1
-            </button>
-        </li>
-        <li class="page-item">
-            <button class="page-link">
-                2
-            </button>
-        </li>
-        <li class="page-item">
-            <button class="page-link">
-                3
-            </button>
-        </li>
-        <li class="page-item">
-            <button class="page-link">
-                🡺
-            </button>
-        </li>
-    </ul>
-</nav>
+    <!-- pagination -->
+    <nav aria-label="Page navigation example">
+        <ul class="pagination d-flex justify-content-center">
+            <li class="page-item disabled">
+                <button class="page-link">
+                    🡸
+                </button>
+            </li>
+            <li class="page-item active"
+                aria-current="page">
+                <button class="page-link">
+                    1
+                </button>
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    2
+                </button>
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    3
+                </button>
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    🡺
+                </button>
+            </li>
+        </ul>
+    </nav>
+</div>
 
 @code {
 
     /// <summary>
     /// List of items to be shown
     /// </summary>
-    [Parameter]
-    public required List<string> Items { get; set; } = Examples;
+    private List<AssetItem> AssetItems { get; set; } = SampleValues;
 
-    // TODO: remove the code for the initialization
-    private static readonly List<string> Examples =
+    private static readonly List<AssetItem> SampleValues =
     [
-        "Trockeneis", "Ethanol", "Natriumhydroxid", "Aceton", "Ammoniumchlorid", "Wasserstoffperoxid",
-        "Kaliumpermanganat", "Messzylinder", "Erlenmeyerkolben", "Reagenzglas"
+        new Chemical
+        {
+            Id = new Guid(), Note = "Temperatur: -78,5\u00b0 C",
+            Room = new Room { Id = new Guid(), Name = "Audimax" }, Name = "Trockeneis", Cas = "124-38-9",
+            ItemNumber = "1845", Shop = "Eisladen", Hazards = [new Hazard { Id = new Guid(), Name = "kalt", ImagePath = "" }],
+            Unit = "kg"
+        },
```

Comment:
📝  Irrelevant here, because it's just test data, but every attribute of the class should be in their own line, if the max line length is exceeded.